app/src/main/java/com/brainwallet/worker/SyncBlockWorker.kt

```diff
@@ -0,0 +1,33 @@
+package com.brainwallet.worker
+
+import android.content.Context
+import androidx.work.Constraints
+import androidx.work.CoroutineWorker
+import androidx.work.NetworkType
+import androidx.work.OneTimeWorkRequestBuilder
+import androidx.work.WorkerParameters
+import com.brainwallet.wallet.BRWalletManager
+
+class SyncBlockWorker(
+    context: Context,
+    params: WorkerParameters
+) : CoroutineWorker(context, params) {
+
+    //todo: maybe need ongoing notification?
+
+    override suspend fun doWork(): Result {
+        BRWalletManager.getInstance().initWallet(applicationContext)
```

Comment:
for now just execute `initWallet` inside this worker

Reply:
Ok. But its still not starting sync until pressing "Ready'

Reply:
already moved to `welcome` https://github.com/gruntsoftware/android/blob/dcd544c9f5b70797d73a8365fa3115f2276af975/app/src/main/java/com/brainwallet/ui/screens/welcome/WelcomeViewModel.kt#L120